### ISSUE TITLE
where to find the results of point cloud?

### ISSUE THREAD

**Issue by Aoihigashi** [KovenYu/WonderJourney#8]
Thanks for sharing your work, I followed the instructions to run the demo, and I can see images and the final saved video in the results, but I couldn't find any files related to the 3D output point cloud in the results folder. Could you please advise where I can find them? Looking forward to your reply.
<img width="198" alt="截屏2024-05-06 22 11 23" src="https://github.com/KovenYu/WonderJourney/assets/130173118/81947d98-1fbb-445a-8bbb-c9e6a9ac4944">

**Comment by Aoihigashi** [comment]
> Thanks for sharing your work, I followed the instructions to run the demo, and I can see images and the final saved video in the results, but I couldn't find any files related to the 3D output point cloud in the results folder. Could you please advise where I can find them? Looking forward to your reply.
> <img alt="截屏2024-05-06 22 11 23" width="198" src="https://private-user-images.githubusercontent.com/130173118/328207663-81947d98-1fbb-445a-8bbb-c9e6a9ac4944.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNTcyNzYsIm5iZiI6MTcxNTE1Njk3NiwicGF0aCI6Ii8xMzAxNzMxMTgvMzI4MjA3NjYzLTgxOTQ3ZDk4LTFmYmItNDQ1YS04YmJiLWM5ZTZhOWFjNDk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwOFQwODI5MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMzIzOWU2YWI5OWE4MzgxY2Q3MTMxZTFiZmE1MTNiYmJjN2NiNDhlMzY5YThlZjgzMGJhNjY1NTRmNGQzOTg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.sLJt1S3I7bHUrHqN2vREbTJXkcJvTPbEjByR_aLhN4M">

already solved.

**Comment by LiRunyi2001** [comment]
Hi there, I also cannot find where to get the output pointcloud file, and could you share how you solved this issue? Thanks in advance!

**Comment by Aoihigashi** [comment]
> Hi there, I also cannot find where to get the output pointcloud file, and could you share how you solved this issue? Thanks in advance!



> Hi there, I also cannot find where to get the output pointcloud file, and could you share how you solved this issue? Thanks in advance!

just uncomment line 334 of the run.py file.
# save_point_cloud_as_ply(torch.cat([kf_interp.points_3d, kf_interp.additional_points_3d], dim=0)*500, kf_interp.run_dir / 'final_point_cloud.ply', torch.cat([kf_interp.kf1_colors, kf_interp.additional_colors], dim=0))
